### PR TITLE
Update Platform Working Group Meeting Schedule

### PR DESCRIPTION
**Pull request by manish-jangra**
This Pull Request will make the following changes -
- Update the platform working group meeting schedule
- Add instructions for signing up to receive calendar invites
- Related GitHub issue - https://github.com/cncf/tag-app-delivery/issues/786

### CODE REVIEW

**Comment by krumware** [comment]
Thanks for doing this! Can you by chance also knock out the changes on the pages for the other languages?

**Comment by manish-jangra** [comment]
> Thanks for doing this! Can you by chance also knock out the changes on the pages for the other languages?

@krumware I don't see the `_index.md` for Spanish and French. Please correct me if I'm wrong. It's been updated for all the other languages.

**Review by abangser — DISMISSED** [review]
> I don't see the `_index.md` for Spanish and French. Please correct me if I'm wrong.Schedule is now updated for all the other languages.

That is correct. There was a change in how we managed translations in the last year where we now generate any non-translated pages. Therefore those two languages, which are our two most recent translations, only have the pages they have actively translated in the repo.

This looks great and thank you for helping us make life easier for the next new joiners!

File: website/content/ko/wgs/platforms/_index.md [hunk 1]
```diff
@@ -23,9 +23,8 @@ To register for updates and to "Join" the group, please also consider joining th
 
 ## 회의
 
-* Meeting schedule: 2nd and 4th Tuesday of each month at [1600 Europe/London time](https://www.timeanddate.com/worldclock/converter.html?iso=20240514T150000&p1=136)
-* Please RSVP for meetings on the [TAG App Delivery Community Groups Page](https://community.cncf.io/tag-app-delivery/). This will also provide a calendar entry.
-* Meeting Location, Virtual: <https://zoom-lfx.platform.linuxfoundation.org/meeting/95319413756?password=a945340e-f322-437f-9653-cbf3f05ada69>
+* Meeting schedule: Every other Tuesday of each month at [1600 Europe/London time](https://www.timeanddate.com/worldclock/converter.html?iso=20240514T150000&p1=136)
+* Join the meeting via [Zoom link](https://zoom-lfx.platform.linuxfoundation.org/meeting/95319413756?password=a945340e-f322-437f-9653-cbf3f05ada69), and You can also add the recurring meetings to your calendar by clicking on the **"Need an invite?"** option on the same page.
```
Comment (SayakMukhopadhyay) [inline website/content/ko/wgs/platforms/_index.md:27]: A nitpick,
```suggestion
* Join the meeting via [Zoom link](https://zoom-lfx.platform.linuxfoundation.org/meeting/95319413756?password=a945340e-f322-437f-9653-cbf3f05ada69). You can also add the recurring meetings to your calendar by clicking on the **"Need an invite?"** option on the same page.
```

File: website/content/en/wgs/platforms/_index.md [hunk 2]
```diff
@@ -23,9 +23,8 @@ To register for updates and to "Join" the group, please also consider joining th
...
 ## Meetings
 
-* Meeting schedule: 2nd and 4th Tuesday of each month at [1600 Europe/London time](https://www.timeanddate.com/worldclock/converter.html?iso=20240514T150000&p1=136)
-* Please RSVP for meetings on the [TAG App Delivery Community Groups Page](https://community.cncf.io/tag-app-delivery/). This will also provide a calendar entry.
-* Meeting Location, Virtual: <https://zoom-lfx.platform.linuxfoundation.org/meeting/95319413756?password=a945340e-f322-437f-9653-cbf3f05ada69>
+* Meeting schedule: Every other Tuesday of each month at [1600 Europe/London time](https://www.timeanddate.com/worldclock/converter.html?iso=20240514T150000&p1=136)
+* Join the meeting via [Zoom link](https://zoom-lfx.platform.linuxfoundation.org/meeting/95319413756?password=a945340e-f322-437f-9653-cbf3f05ada69), and You can also add the recurring meetings to your calendar by clicking on the **"Need an invite?"** option on the same page.
   * Passcode: 77777
-* Agendas and notes: <https://docs.google.com/document/d/1_smeS9-j-SuHJi0VXjx4g9xiD2-tgqhnlwf5oSMDQgg>
+* Agendas and Notes can be found on this [Google Doc](https://docs.google.com/document/d/1_smeS9-j-SuHJi0VXjx4g9xiD2-tgqhnlwf5oSMDQgg)
```
Comment (SayakMukhopadhyay) [inline website/content/en/wgs/platforms/_index.md:29]: Another nitpick
```suggestion
* Agendas and notes can be found in this [Google Doc](https://docs.google.com/document/d/1_smeS9-j-SuHJi0VXjx4g9xiD2-tgqhnlwf5oSMDQgg)
```

File: website/content/ja/wgs/platforms/_index.md [hunk 3]
```diff
@@ -23,9 +23,8 @@ To register for updates and to "Join" the group, please also consider joining th
...
 ## Meetings
 
-* Meeting schedule: 2nd and 4th Tuesday of each month at [1600 Europe/London time](https://www.timeanddate.com/worldclock/converter.html?iso=20240514T150000&p1=136)
-* Please RSVP for meetings on the [TAG App Delivery Community Groups Page](https://community.cncf.io/tag-app-delivery/). This will also provide a calendar entry.
-* Meeting Location, Virtual: <https://zoom-lfx.platform.linuxfoundation.org/meeting/95319413756?password=a945340e-f322-437f-9653-cbf3f05ada69>
+* Meeting schedule: Every other Tuesday of each month at [1600 Europe/London time](https://www.timeanddate.com/worldclock/converter.html?iso=20240514T150000&p1=136)
+* Join the meeting via [Zoom link](https://zoom-lfx.platform.linuxfoundation.org/meeting/95319413756?password=a945340e-f322-437f-9653-cbf3f05ada69), and You can also add the recurring meetings to your calendar by clicking on the **"Need an invite?"** option on the same page.
   * Passcode: 77777
-* Agendas and notes: <https://docs.google.com/document/d/1_smeS9-j-SuHJi0VXjx4g9xiD2-tgqhnlwf5oSMDQgg>
+* Agendas and Notes can be found on this [Google Doc](https://docs.google.com/document/d/1_smeS9-j-SuHJi0VXjx4g9xiD2-tgqhnlwf5oSMDQgg)
```
Comment (SayakMukhopadhyay) [inline website/content/ja/wgs/platforms/_index.md:29]: Another nitpick
```suggestion
* Agendas and notes can be found in this [Google Doc](https://docs.google.com/document/d/1_smeS9-j-SuHJi0VXjx4g9xiD2-tgqhnlwf5oSMDQgg)
```

File: website/content/ko/wgs/platforms/_index.md [hunk 4]
```diff
@@ -23,9 +23,8 @@ To register for updates and to "Join" the group, please also consider joining th
...
   * Passcode: 77777
-* Agendas and notes: <https://docs.google.com/document/d/1_smeS9-j-SuHJi0VXjx4g9xiD2-tgqhnlwf5oSMDQgg>
+* Agendas and Notes can be found on this [Google Doc](https://docs.google.com/document/d/1_smeS9-j-SuHJi0VXjx4g9xiD2-tgqhnlwf5oSMDQgg)
```
Comment (SayakMukhopadhyay) [inline website/content/ko/wgs/platforms/_index.md:29]: Another nitpick
```suggestion
* Agendas and notes can be found in this [Google Doc](https://docs.google.com/document/d/1_smeS9-j-SuHJi0VXjx4g9xiD2-tgqhnlwf5oSMDQgg)
```

File: website/content/zh/wgs/platforms/_index.md [hunk 5]
```diff
@@ -23,9 +23,8 @@ To register for updates and to "Join" the group, please also consider joining th
...
 ## 会议
 
-* Meeting schedule: 2nd and 4th Tuesday of each month at [1600 Europe/London time](https://www.timeanddate.com/worldclock/converter.html?iso=20240514T150000&p1=136)
-* Please RSVP for meetings on the [TAG App Delivery Community Groups Page](https://community.cncf.io/tag-app-delivery/). This will also provide a calendar entry.
-* Meeting Location, Virtual: <https://zoom-lfx.platform.linuxfoundation.org/meeting/95319413756?password=a945340e-f322-437f-9653-cbf3f05ada69>
+* Meeting schedule: Every other Tuesday of each month at [1600 Europe/London time](https://www.timeanddate.com/worldclock/converter.html?iso=20240514T150000&p1=136)
+* Join the meeting via [Zoom link](https://zoom-lfx.platform.linuxfoundation.org/meeting/95319413756?password=a945340e-f322-437f-9653-cbf3f05ada69), and You can also add the recurring meetings to your calendar by clicking on the **"Need an invite?"** option on the same page.
   * Passcode: 77777
-* Agendas and notes: <https://docs.google.com/document/d/1_smeS9-j-SuHJi0VXjx4g9xiD2-tgqhnlwf5oSMDQgg>
+* Agendas and Notes can be found on this [Google Doc](https://docs.google.com/document/d/1_smeS9-j-SuHJi0VXjx4g9xiD2-tgqhnlwf5oSMDQgg)
```
Comment (SayakMukhopadhyay) [inline website/content/zh/wgs/platforms/_index.md:29]: Another nitpick
```suggestion
* Agendas and notes can be found in this [Google Doc](https://docs.google.com/document/d/1_smeS9-j-SuHJi0VXjx4g9xiD2-tgqhnlwf5oSMDQgg)
```

File: platforms-wg/README.md [hunk 6]
```diff
@@ -19,11 +19,10 @@ To register for updates and to "Join" the group, please also consider joining th
 
 ## Meetings
 
-* Meeting schedule: 2nd and 4th Tuesday of each month at [1600 Europe/London time](https://www.timeanddate.com/worldclock/converter.html?iso=20240514T150000&p1=136)
-* Please RSVP for meetings on the [TAG App Delivery Community Groups Page](https://community.cncf.io/tag-app-delivery/). This will also provide a calendar entry.
-* Meeting Location, Virtual: <https://zoom-lfx.platform.linuxfoundation.org/meeting/95319413756?password=a945340e-f322-437f-9653-cbf3f05ada69>
+* Meeting schedule: Every other Tuesday of each month at [1600 Europe/London time](https://www.timeanddate.com/worldclock/converter.html?iso=20240514T150000&p1=136)
+* Join the meeting via [Zoom link](https://zoom-lfx.platform.linuxfoundation.org/meeting/95319413756?password=a945340e-f322-437f-9653-cbf3f05ada69), and You can also add the recurring meetings to your calendar by clicking on the **"Need an invite?"** option on the same page.
```
[the remark this answers is not in the document]
Comment (manish-jangra) [reply platforms-wg/README.md]: incorportaed suggested changes.

File: website/content/en/wgs/platforms/_index.md [hunk 7]
```diff
@@ -23,9 +23,8 @@ To register for updates and to "Join" the group, please also consider joining th
 
 ## Meetings
 
-* Meeting schedule: 2nd and 4th Tuesday of each month at [1600 Europe/London time](https://www.timeanddate.com/worldclock/converter.html?iso=20240514T150000&p1=136)
-* Please RSVP for meetings on the [TAG App Delivery Community Groups Page](https://community.cncf.io/tag-app-delivery/). This will also provide a calendar entry.
-* Meeting Location, Virtual: <https://zoom-lfx.platform.linuxfoundation.org/meeting/95319413756?password=a945340e-f322-437f-9653-cbf3f05ada69>
+* Meeting schedule: Every other Tuesday of each month at [1600 Europe/London time](https://www.timeanddate.com/worldclock/converter.html?iso=20240514T150000&p1=136)
+* Join the meeting via [Zoom link](https://zoom-lfx.platform.linuxfoundation.org/meeting/95319413756?password=a945340e-f322-437f-9653-cbf3f05ada69), and You can also add the recurring meetings to your calendar by clicking on the **"Need an invite?"** option on the same page.
```
[the remark this answers is not in the document]
Comment (manish-jangra) [reply website/content/en/wgs/platforms/_index.md]: incorportaed suggested changes.

File: website/content/ja/wgs/platforms/_index.md [hunk 8]
```diff
@@ -23,9 +23,8 @@ To register for updates and to "Join" the group, please also consider joining th
 
 ## Meetings
 
-* Meeting schedule: 2nd and 4th Tuesday of each month at [1600 Europe/London time](https://www.timeanddate.com/worldclock/converter.html?iso=20240514T150000&p1=136)
-* Please RSVP for meetings on the [TAG App Delivery Community Groups Page](https://community.cncf.io/tag-app-delivery/). This will also provide a calendar entry.
-* Meeting Location, Virtual: <https://zoom-lfx.platform.linuxfoundation.org/meeting/95319413756?password=a945340e-f322-437f-9653-cbf3f05ada69>
+* Meeting schedule: Every other Tuesday of each month at [1600 Europe/London time](https://www.timeanddate.com/worldclock/converter.html?iso=20240514T150000&p1=136)
+* Join the meeting via [Zoom link](https://zoom-lfx.platform.linuxfoundation.org/meeting/95319413756?password=a945340e-f322-437f-9653-cbf3f05ada69), and You can also add the recurring meetings to your calendar by clicking on the **"Need an invite?"** option on the same page.
```
[the remark this answers is not in the document]
Comment (manish-jangra) [reply website/content/ja/wgs/platforms/_index.md]: incorportaed suggested changes.

File: website/content/zh/wgs/platforms/_index.md [hunk 9]
```diff
@@ -23,9 +23,8 @@ To register for updates and to "Join" the group, please also consider joining th
 
 ## 会议
 
-* Meeting schedule: 2nd and 4th Tuesday of each month at [1600 Europe/London time](https://www.timeanddate.com/worldclock/converter.html?iso=20240514T150000&p1=136)
-* Please RSVP for meetings on the [TAG App Delivery Community Groups Page](https://community.cncf.io/tag-app-delivery/). This will also provide a calendar entry.
-* Meeting Location, Virtual: <https://zoom-lfx.platform.linuxfoundation.org/meeting/95319413756?password=a945340e-f322-437f-9653-cbf3f05ada69>
+* Meeting schedule: Every other Tuesday of each month at [1600 Europe/London time](https://www.timeanddate.com/worldclock/converter.html?iso=20240514T150000&p1=136)
+* Join the meeting via [Zoom link](https://zoom-lfx.platform.linuxfoundation.org/meeting/95319413756?password=a945340e-f322-437f-9653-cbf3f05ada69), and You can also add the recurring meetings to your calendar by clicking on the **"Need an invite?"** option on the same page.
```
[the remark this answers is not in the document]
Comment (manish-jangra) [reply website/content/zh/wgs/platforms/_index.md]: incorportaed suggested changes.

File: website/content/en/wgs/platforms/_index.md (hunk 2 above)
Comment (manish-jangra) [reply website/content/en/wgs/platforms/_index.md]: incorportaed suggested changes.

File: website/content/ja/wgs/platforms/_index.md (hunk 3 above)
Comment (manish-jangra) [reply website/content/ja/wgs/platforms/_index.md]: incorportaed suggested changes.

File: website/content/zh/wgs/platforms/_index.md (hunk 5 above)
Comment (manish-jangra) [reply website/content/zh/wgs/platforms/_index.md]: incorportaed suggested changes.

**Review by SayakMukhopadhyay — APPROVED** [review]

**Review by krumware — APPROVED** [review]
lgtm

**Review by GenPage — APPROVED** [review]
LGTM

**Review by roberthstrand — APPROVED** [review]

**Pull request merged**